File: CPP/Aksesoris.cpp
#include <iostream>
#include <string>
#include <list>
#include "Petshop.cpp"
using namespace std;

class Aksesoris : public Petshop {

    private:
        string jenis;
        string bahan;
        string warna;
    public:

        Aksesoris(){
            jenis = "";
            bahan = "";
            warna = "";
        }

        Aksesoris(string jenis, string bahan, string warna){
            this->jenis = jenis;
            this->bahan = bahan;
            this->warna = warna;
        }

        void setJenis(string jenis){
            this->jenis = jenis;
        }

        void setBahan(string bahan){
            this->bahan = bahan;
        }

        void setWarna(string warna){
            this->warna = warna;
        }

        string getJenis(){
            return jenis;
        }

        string getBahan(){
            return bahan;
        }

        string getWarna(){
            return warna;
        }
        ~Aksesoris(){
        }
};

File: PHP/Aksesoris.php
<?php
include 'Petshop.php';

class Aksesoris extends Petshop {
    private $jenis = "";
    private $bahan = "";
    private $warna = "";

    //constructor
    public function __construct($id, $nama, $harga, $stok, $gambar, $jenis, $bahan, $warna) {
        parent::__construct($id, $nama, $harga, $stok, $gambar);
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }
    //getter setter
    public function getJenis() {
        return $this->jenis;
    }

    public function setJenis($jenis) {
        $this->jenis = htmlspecialchars($jenis);
    }

    public function getBahan() {
        return $this->bahan;
    }

    public function setBahan($bahan) {
        $this->bahan = htmlspecialchars($bahan);
    }

    public function getWarna() {
        return $this->warna;
    }

    public function setWarna($warna) {
        $this->warna = htmlspecialchars($warna);
    }



}
?>